### DISCUSSION TITLE
I want q-input :rules to expand with transition as rule is applied

### DISCUSSION THREAD

**Discussion by joshhhso** [quasarframework/quasar#12918] [General - Components / Directives / etc] [answered]
Hi. I've been using quasar for several month now.
I'm having trouble in making list of input tags. But I wanted them to be lined up nicely.
However, when I apply :rules on q-input, it seems like it automatically gives padding-bottom 20px. 
I naturally thought It would expand as rule is applied... I couldn't find any possible solution by myself...
Hope that anyone could help me on this one! 

![Screen Shot 2022-03-24 at 8 55 39 PM](https://user-images.githubusercontent.com/79290170/159911147-2c7d3a15-13dd-43bd-afdc-b12220c988cc.png)

FYI. please let me know my explanation doesn't make sense. sorry for my poor english :(

**Comment by MilosPaunovic** [comment, marked as answer]
Use `hide-bottom-space` prop of `QInput`.
